documentation/book/con-certificate-renewal.adoc

```diff
@@ -35,24 +33,22 @@ The behavior of the Cluster Operator during the renewal period depends on whethe
 
 The Cluster Operator performs the following process to renew CA certificates:
 
-. Generate a new CA key and certificate. The new private key replaces the old private key in the corresponding `Secret`. The new certificate is given the name `ca.crt` within the corresponding `Secret` and the old certificate is renamed `ca-_<expiry-date>_.crt`.
+. Generate a new CA certificate, but retaining the existing key. The new certificate replaces the old one with the name `ca.crt` within the corresponding `Secret`.
 
-. Restart Zookeeper nodes so that they will trust the new CA certificate.
+. Generate new client certificates (for Zookeeper nodes, Kafka brokers, and the entity operator).
+This is not strictly necessary because the signing key has not changed, but it keeps the validity period of the client certificate in sync with the CA certificate.
 
-. Restart Kafka brokers so that they will trust the new CA certificate.
+. Restart Zookeeper nodes so that they will trust the new CA certificate and use the new client certificates.
 
-. Restart the Topic and User Operators so that they will trust the new CA certificate.
+. Restart Kafka brokers so that they will trust the new CA certificate and use the new client certificates.
 
-. Generate new client certificates (for Zookeeper nodes, Kafka brokers, and the entity operator) signed by the new CA.
+. Restart the Topic and User Operators so that they will trust the new CA certificate and use the new client certificates.
 
-. Perform the same restarts so that clients are using certificates signed by the new CA certificate.
-
-At the end of the renewal period the Cluster Operator will remove the now expired CA certificates (those named `ca-_<expiry-date>_.crt`) from the corresponding `Secret` and perform a further round of restarts.
 
 == Renewal process with your own CA certificates
 
-At the start of the renewal period the Cluster Operator will start logging at the `WARN` level that new CA certificates and keys are needed.
-Once you have provided the new certificates and keys, the Cluster Operator performs a further set of restarts within the Kafka cluster for which the warning was issued.
+At the start of the renewal period the Cluster Operator will start logging at the `WARN` level that new a CA certificate is needed.
+Once you have provided the new certificate, the Cluster Operator performs a further set of restarts within the Kafka cluster for which the warning was issued.
```

Comment:
Just  a doubt about this paragraph. You commented `proc-renewing-your-own-ca-certificates.adoc` but now we are leaving here how the renewal process works with your own CA certificates. I was wondering that if people read this, they could ask ... how to do that? and then the proc is needed.

Reply:
You're right. I guess I should just take this section out for the time being then.

Reply:
Agree